connector_prestashop/unit/importer.py

```diff
@@ -492,13 +496,18 @@ def _after_import(self, binding):
 def import_batch(session, model_name, backend_id, filters=None, **kwargs):
     """ Prepare a batch import of records from PrestaShop """
     env = get_environment(session, model_name, backend_id)
-    importer = env.get_connector_unit(BatchImporter)
-    importer.run(filters=filters, **kwargs)
+    ctx = dict(session.context, **kwargs)
+    with env.session.change_context(ctx):
```

Comment:
Why do you set the values in the context if you pass them to the importer kwargs?